src/index.ts

```diff
@@ -1,5 +1,2 @@
 export { createIsolation } from './createIsolation';
-export {
-  ScopeContext as INTERNAL_ScopeContext,
```

Comment:
@dai-shi Do we need this? We should remove it if its no longer needed.

Reply:
I think it's requested by somebody.

Reply:
I'd prefer to remove it. This isn't in the docs and the api / behavior here will likely change.

Reply:
That's why it's prefixed `INTERNAL_`. It implies no docs and breaking changes.
Can you check the commit history and see who requested it?

Reply:
commit: e6f7a9f350595a10419a3e161c7f716f393f0d40
```feat(scope): patch store to avoid patching hooks ([#9](https://github.com/dmaskasky/jotai-scope/issues/9))

* feat(scope): patch store to avoid patching hooks

* expose internal scope context```

https://github.com/dmaskasky/jotai-scope/issues/9 (404 for me)

Reply:
It's https://github.com/jotaijs/jotai-scope/pull/9.
So, I think if @yf-yang agrees, we can drop it.

Reply:
I agree to drop it as I don't know what's the specific use case, too. Maybe we tried to explicitly use that in some csb examples before? Can't remember that.